### ISSUE TITLE
podcast:discuss, or attribute for podcast:social?

### ISSUE THREAD

**Issue by bjoreman** [Podcastindex-org/podcast-namespace#153]
Many podcasts have a chat room, discussion forum, Slack group, Discord server, or some other main point of interaction. It would be useful to be able to point that out in a standard way so that playback apps can display a nice big "Talk to us!"-button or, when possible, even embed the discussion.

My first thought was a podcast:discuss tag. It would have a single value and could appear on the feed itself as well as individual episodes. (So that episodes could link to their own thread within a main forum, for example.) Then, I started wondering if it would make sense to use some attribute (default?) on the podcast:social tag instead. "Here's a list of ways of interacting with the show, and this is the recommended one."

A further question is if there should be attributes to inform an app whether it's even worth trying to embed or not. On one hand, a join link for Discord or Slack may not be worth embedding. On the other, the ability of app developers to make stuff work should not be underestimated.

**Comment by daveajones** [comment]
Do you know if there is prior art anywhere on this?  I know that the NA chat room isn’t being referenced in any structural way in the feed. I wonder about other shows.

To my knowledge I’ve never seen anything like that.  It’s a good idea.

**Comment by bjoreman** [comment]
No, I have seen nothing like it either :-) … But the idea popped up when I thought of the payment icon Overcast adds for podcasts with the payment in link in their feed.

**Comment by jamescridland** [comment]
I love this idea.

Is there anything in the existing pingback/trackback standard in RSS that we can use?

https://make.wordpress.org/support/user-manual/building-your-wordpress-community/trackbacks-and-pingbacks/

The idea of interacting with a show is a great one. I'd love to see it be, somehow, decentralised, so that comments can be made in any supporting app and be visible in any supporting app. That would be most excellent.

**Comment by theDanielJLewis** [comment]
I see two options.

Community: this is a channel level tag for the podcast’s community (or maybe more than one), such as a Facebook group, Slack team, Discord server, IRC, etc.

Discussion: this seems more appropriate for an episode tag to discuss the content of that episode. It could link to a web page with comments, a forum thread, or even a prepopulated hashtag.

The difficult becomes supporting when there are multiple places for each.

And I think Community is different from Social. Whereas Social would be where you can follow the “broadcasts” of a podcast profile, Community is where you can start your own conversations or join those with other members.

**Comment by brianoflondon** [comment]
Love the Community tag idea.

Also Love the Discussion tag to be used for discussions about a specific episode.

Consider this a simple vote of support for both.

**Comment by douglaskastle** [comment]
I like this.  It does remind me a little bit of what happened in the blogging/commenting space.  Wordpress has it's own commenting system, however it is a pain to moderate and/or self moderate.  So what a lot of bloggers, did is use a dedicated plugin in that hosted the comments elsewhere, like disqus.  They provide a service that handles user identification/authentication and in some cases moderation and remove the load from the self hoster. Using a service like this though leaves you open to being de-platformed.

I am unsure is there is a similar service for podcasting, arguably if you have the self hosted podcast website, which a post per episode, there is no reason you couldn't boot strap with that.  

My feeling that is that the moderation of any user contributed system is the long term block for things like this, this also goes for a more open podcast review system, but that I think is a different conversation.

(I acknowledge I am showing my age by using the phrase "blogger")

**Comment by theDanielJLewis** [comment]
> I am unsure is there is a similar service for podcasting, arguably if you have the self hosted podcast website, which a post per episode, there is no reason you couldn't boot strap with that.

My idea for the `discussion` tag would be simply the URL where that discussion takes place, not a technology to support it.

So, for example, I might link the discussion to the comments section of my website, the Facebook post where I shared the episode, or a forum thread.

However, I realized this `discussion` tag might be unnecessary, as we already have `<comments>` [in the RSS spec](https://cyber.harvard.edu/rss/rss.html#ltcommentsgtSubelementOfLtitemgt) ([also here](https://cyber.harvard.edu/rss/weblogComments.html)). For example:

```xml
<comments>https://theaudacitytopodcast.com/labeling-podcasters-amateur-vs-skilled/#respond</comments>
```

**Comment by daveajones** [comment]
I want to put this idea into Phase 3.  An open protocol like Matrix was proposed on podcastindex.social , and I think that makes sense.  I'll write up an initial tag proposal so we have a base to work with and argue over.

**Comment by stuartjmoore** [comment]
What separates “social” from “discuss” or even a plain webpage?

**Comment by douglaskastle** [comment]
Has any one setup a matrix.org channel for either podcast.social and/or podcasting 2.0?  It would be nice to demo on what it actually would be to understand how it might work.

Is there a channel per podcast or is it a channel per episode or is it some thing else? I am curious to see this in the wild.

**Comment by benjaminbellamy** [comment]
In order to allow Podfriend and Castopod users to interact together using ActivityPub, I wrote a proposal for a podcast:social spec: https://github.com/benjaminbellamy/podcast-namespace/blob/patch-9/proposal-docs/social/social.md
This spec tries to cover all social networks (Facebook, Instagram, Twitter, Mastodon…) and discussion platforms (Slack, Discord, Jabber, IRC, Matrix…) and not only ActivityPub.

**Comment by Marzal** [comment]
We now have:
 - https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/social-interact.md - [permalink](https://github.com/Podcastindex-org/podcast-namespace/blob/336978220c14c0f180282ee0feb3e027567c4522/docs/tags/social-interact.md)
- https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/chat.md - [permalink](https://github.com/Podcastindex-org/podcast-namespace/blob/336978220c14c0f180282ee0feb3e027567c4522/docs/tags/chat.md)

And the `<comments>` from the RSS always was there too.

Some related Pull Request: https://github.com/Podcastindex-org/podcast-namespace/pull/311

I think that cover most of the proposal. Please open a new [discussion](https://github.com/Podcastindex-org/podcast-namespace/discussions) if not.
